Update release notes for v3.13.0 release (#2252)
alphagov/govuk-frontend: CHANGELOG.md

## Unreleased

## 3.13.0 (Feature release)

### New features

#### Add a 'none' option and 'or' divider to checkboxes

You can now add a 'none' option to checkboxes. For example, 'None of the above' or 'None of these options apply to me'.

Use the 'none' option to [allow users to state that none of the checkboxes apply to them](https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none-). Otherwise, users would have to work out that they should leave all the checkboxes unticked.

You can now also add an 'or' divider before the 'none' option. Use the 'or' divider to make the 'none' option more visible to users.

> This was added in [pull request #2151: Add 'None of the above' and 'or' divider to checkboxes](https://github.com/alphagov/govuk-frontend/pull/2151).

Reviewer: ```suggestion
This was added in [pull request #2151: Add 'None of the above' and 'or' divider to checkboxes](https://github.com/alphagov/govuk-frontend/pull/2151). Thanks to @frankieroberto for contributing this improvement.
```

Reviewer 2: We should make the link explicit as we do with `@natcarey` below:

```suggestion
This was added in [pull request #2151: Add 'None of the above' and 'or' divider to checkboxes](https://github.com/alphagov/govuk-frontend/pull/2151). Thanks to [@frankieroberto](https://github.com/frankieroberto] for contributing this improvement.
```

Reviewer: Thanks @lfdebrux ⬆️  looks good I'm happy to commit it.

## Unreleased

## 3.13.0 (Feature release)

### New features

#### Add a 'none' option and 'or' divider to checkboxes

> You can now add a 'none' option to checkboxes. For example, 'None of the above' or 'None of these options apply to me.'

Reviewer 3: 'None of these options apply to me.' => 'None of these options apply to me'.

Author: ```suggestion
You can now add a 'none' option to checkboxes. For example, 'None of the above' or 'None of these options apply to me'.
```

> Use the 'none' option to [allow users to state that none of the checkboxes apply to them](https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none-). Otherwise, users would have to work out that they should leave all the checkboxes unticked.

Reviewer 3: Is there going to be content in this page on adding an option for none? I don't see any on this link at the moment

Author: There will be content on the page for this option, but our content designer is still finalising it. By the time these release notes go out, the content will be visible under the 'Add an option for none' heading.

> You can now also add an 'or' divider before the 'none' option. Use the 'or' divider to make the 'none' option stand out from the other options so it's more visible to users.

Reviewer 3: You could probably simplify. For example: 

- "Use the 'or' divider to make the 'none' option more visible to users"
- "Use the 'or' divider to make the 'none' option stand out from the other options"

Author: ```suggestion
You can now also add an 'or' divider before the 'none' option. Use the 'or' divider to make the 'none' option more visible to users.
```

This was added in [pull request #2245: Allow the CSP `nonce` attribute to be set on the inline script in the page template](https://github.com/alphagov/govuk-frontend/pull/2245). Thanks to [@natcarey](https://github.com/natcarey) for contributing this issue and its solution.

> #### Add a 'none' option and 'or' divider to checkboxes

Reviewer 4: This looks really good to me – my only suggestion is that we introduce this feature first. The ability to add nonces to scripts is going to be relevant for a very small number of users, so this is really the 'headline' feature.

Author: Thanks - have reordered so that the checkboxes content displays first.

You can now add a 'none' option to checkboxes. For example, 'None of the above' or 'None of these options apply to me.'

> Use the 'none' option to [allow users to state that none of the checkboxes apply to them](https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none). Otherwise, users would have to work out that they should leave all the checkboxes unticked.

Author: Basing the link on the subheading in line 68 of the [pull request for the content](https://github.com/alphagov/govuk-design-system/pull/1535/files).

Reviewer 4: Unfortunately I think there's a bug in the code that turns the headings into anchor links. Because of the quote after 'none' (I assume) the actual anchor link has a stray `-` at the end which will need including here too.

```suggestion
Use the 'none' option to [allow users to state that none of the checkboxes apply to them](https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none-). Otherwise, users would have to work out that they should leave all the checkboxes unticked.
```

You can now also add an 'or' divider before the 'none' option. Use the 'or' divider to make the 'none' option stand out from the other options so it's more visible to users.

This was added in [pull request #2151: Add 'None of the above' and 'or' divider to checkboxes](https://github.com/alphagov/govuk-frontend/pull/2151).

### Fixes

We've made the following fixes in [pull request #2229: Change approach to fallback PNG in the header to fix blank data URI from triggering Content Security Policy error](https://github.com/alphagov/govuk-frontend/pull/2229):

- fix blank data URI from triggering CSP error
- fix alignment of fallback PNG in the header

We’ve also made fixes in the following pull requests:

- [#2228: Fix display of checkboxes in Internet Explorer 8](https://github.com/alphagov/govuk-frontend/pull/2228)
- [#2237: Fix GOV.UK logo disappearing on light background in Windows High Contrast Mode](https://github.com/alphagov/govuk-frontend/pull/2237)

> - [#2151: Avoid ink skipping on link hover state underlines](https://github.com/alphagov/govuk-frontend/pull/2251)

Reviewer 4: ```suggestion
- [#2251: Avoid ink skipping on link hover state underlines](https://github.com/alphagov/govuk-frontend/pull/2251)
```
